command_test.go

```diff
@@ -1099,6 +1099,18 @@ func TestShorthandVersionTemplate(t *testing.T) {
 	checkStringContains(t, output, "customized version: 1.0.0")
 }
 
+func TestErrPrefix(t *testing.T) {
```

Comment:
Can you add two more tests:
1. only setting an error prefix on the root, but triggerring it by a sub command to see that it still gets printed
2. setting a different error prefix for a sub command and checking it gets printed when the sub command fails

Reply:
- When I generate an object with `&Command{}`, is `Run: emptyRun` needed?
- What names do you think are suitable for functions 1 and 2?

Reply:
> * When I generate an object with `&Command{}`, is `Run: emptyRun` needed?

you probably need it for the deepest child command but it’s ok to have it for every command

> * What names do you think are suitable for functions 1 and 2?

You mean for the sub command?
How about child1 and child2?

Reply:
Thank you!

I mean 2 test functions, `TestRootErrPrefixExecutedOnSubcommand()` and `TestRootAndSubErrPrefix()` temporarily named are.

Reply:
Oh that makes more sense😁
Those names are fine